utils/run_test.sh

```diff
@@ -52,6 +52,26 @@ run_test()
     fi
 
     ((log_num += 1))
+
+    FILES="${DAOS_BASE}/test_results/*.xml"
+    for f in $FILES
```

Comment:
Only if you have to rebase.
Minor: May be a better practice will be to use complete words than single letter variables.
`for file in $FILES` instead of `for f in $FILES`
And same changes wherever 'f' was being used.

Reply:
ok. will change in next commit.